src/sample/View.java

```diff
@@ -68,28 +73,25 @@ private void setupGUI()
     // setting up the start screen
     void startScreen(Stage startGUI)
     {
+        // gap between elements in the start screen
         this.gridPane.setHgap(8);
         this.gridPane.setVgap(8);
 
+        // elements for the start screen gui stage
         Text handle = new Text("Welcome to the Random Bot Game!");
-        handle.setTextAlignment(TextAlignment.CENTER);
-        Text argInquire = new Text("What is the Grid size?");
-        argInquire.setTextAlignment(TextAlignment.CENTER);
+        Text argInquire = new Text("What is the Grid size? (size <= 15)");
         Button submitStart = new Button("Go!");
-        submitStart.setTextAlignment(TextAlignment.CENTER);
         CheckBox defaultBehaviors = new CheckBox("Defaults for the application");
-        defaultBehaviors.setTextAlignment(TextAlignment.CENTER);
         TextField gridSizeStart = new TextField();
 
-            submitStart.setOnAction(actionEvent -> prepTheGame(argInquire, defaultBehaviors,
+        // button action to start the actual game
+        submitStart.setOnAction(actionEvent -> prepTheGame(argInquire, defaultBehaviors,
```

Comment:
Don't we have a method where we initialize the variable actions?  We should probably follow that pattern for something like this too.

Reply:
I was having difficulty bringing methods with arguments into that pattern but yeah if that is doable then feel free to either advise on how or show me how. I was trying to bring in arguments to the consumer pattern as all the current ones are null args and it kept red lining.